extensions/blocks/conversation/edit.js

```diff
@@ -0,0 +1,151 @@
+/**
+ * External dependencies
+ */
+import { filter, map } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import {
+	useEffect,
+	useRef,
+	useCallback,
+	useMemo,
+} from '@wordpress/element';
+import {
+	InnerBlocks,
+	InspectorControls,
+	BlockControls,
+} from '@wordpress/block-editor';
+import {
+	Panel,
+	PanelBody,
+	ToggleControl,
+	ToolbarGroup,
+} from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import './editor.scss';
+import ParticipantsDropdown, { ParticipantsSelector } from './components/participants-controls';
+import TranscritptionContext from './components/context';
+
+import { list as defaultParticipants } from './participants.json';
+
+const TRANSCRIPTION_TEMPLATE = [
+	[ 'core/heading', { placeholder: __( 'Conversation title', 'jetpack' ) } ],
+	[ 'jetpack/dialogue', defaultParticipants[ 0 ] ],
+	[ 'jetpack/dialogue', defaultParticipants[ 1 ] ],
+	[ 'jetpack/dialogue', defaultParticipants[ 2 ] ],
+];
+
+function ConversationEdit ( {
+	className,
+	attributes,
+	setAttributes,
+} ) {
+	const { participants, showTimestamps, className: classNameAttr } = attributes;
+	const containertRef = useRef();
```

Comment:
Nitpick and non-blocking: Seems to be a typo here.
```suggestion
	const containerRef = useRef();
```